new/LIT6725Letter.xml

```diff
@@ -0,0 +1,81 @@
+    <teiHeader>
+        <fileDesc>
+            <titleStmt>
+                <title xml:lang="en" xml:id="t1">Letter from Patriarch John XVII to King ʾIyāsu II</title>
+                <editor role="generalEditor" key="AB"/>
+                <editor key="DR"/>
+                <funder>Akademie der Wissenschaften in Hamburg</funder>
+            </titleStmt>
+            <publicationStmt>
+                <authority>Hiob-Ludolf-Zentrum für Äthiopistik</authority>
+                <publisher>Die Schriftkultur des christlichen Äthiopiens und Eritreas: Eine multimediale
+                            Forschungsumgebung / Beta maṣāḥǝft</publisher>
+                <pubPlace>Hamburg</pubPlace>
+                <availability>
+                    <licence target="http://creativecommons.org/licenses/by-sa/4.0/"> This file is
+                                licensed under the Creative Commons Attribution-ShareAlike 4.0. </licence>
+                </availability>
+            </publicationStmt>
+            <sourceDesc>
+                <p/>
+            </sourceDesc>
+        </fileDesc>
+        <encodingDesc>
+            <p>A digital born TEI file</p>
+        </encodingDesc>
+        <profileDesc>
+            <creation when="1743"/>
+            <abstract>
+                <p>A letter from Patriarch <persName ref="PRS5712JohnXVI"/> to King <persName ref="PRS5617IyasuII"/>,
+                    dated to 1743,
+                    concerning mostly the construction of the church of <placeName ref="LOC6533DabraSahayQwe"/>. The letter is attested in 
+                    <ref type="mss" corresp="BLorient481"/> and <ref type="mss" corresp="BLDaethb3"/>.
+                </p>
+            </abstract>
+            <textClass>
+                <keywords>
+                    <term key="ChristianLiterature"/>
+                    <term key="Letter"/>
+                </keywords>
+            </textClass>
+            <langUsage>
+                <language ident="en">English</language>
+                <language ident="gez">Gǝʿǝz</language>
+            </langUsage>
+        </profileDesc>
+        <revisionDesc>
+            <change who="DR" when="2022-09-26">Created entity</change>
+        </revisionDesc>
+    </teiHeader>
+    <text>
+        <body>
+          <div type="edition">
+              <note>The following excerpts are taken from <ref type="mss" corresp="BLDaethb3"/>.</note>
+              <div type="textpart" subtype="incipit" xml:lang="gez">
+                  <ab>
+                      መጽሐፈ፡ መልእክት፡ ዘተፈነወት፡ እምዮሐንስ፡ ገብረ፡ እግዚአብሔር፡ ዘተሰምየ፡ ዘጸጋሁ፡ ወተሠይመ፡ <add place="overstrike">በፈቃዱ፡</add> ለተልእኮ፡ መንበር፡ ማርቆሳዊ፡ በሀገረ፡ ዓባይ፡
+                      እለ፡ እስክንድርያ፡ ወትዕይንተ፡ ኢየሩሳሌም፡ ወገዳማተ፡ ምስር፡ ወበሐውርተ፡ ኢትዮጵያ፡ ሰላመ፡ እግዚአብሔር፡ ቅዱስ፡ ሕይወተ፡ ሥጋ፡ ወኵሉ፡ ነፍስ፡ ይኀድር፡ ዝንቱ፡
+                      ሰላም፡ መንፈሳዊ፡ ላዕለ፡ ውሉድነ፡ ውሉደ፡ መንግሥት፡ ልዑላን፡ በሥልጣን፡ ወዕበይ፡ ወክብር፡ ወዓዲ፡ ይኀድር፡ ጸጋ፡ <add place="overstrike">አ</add>ምላካዊ፡ ወበረከት፡ ሰማያዊ፡ ላዕሌከ፡ ኦወልድነ፡ 
```

Comment:
Just one remark: ይኀድር might be marked with `sic` (the standard spelling is የኀድር), but if you think that this is too much zeal on our side, please ignore my suggestion